diff --git a/chrome/browser/permissions/permission_manager.cc b/chrome/browser/permissions/permission_manager.cc
index 4ac3f55f730cb08c1e4031369f1a07053bdcc044..225fd96cf89ac0af3a39546f26afd766192eb300 100644
--- a/chrome/browser/permissions/permission_manager.cc
+++ b/chrome/browser/permissions/permission_manager.cc
@@ -70,6 +70,7 @@ PermissionStatus ContentSettingToPermissionStatus(ContentSetting setting) {
     case CONTENT_SETTING_ASK:
       return PermissionStatus::ASK;
     case CONTENT_SETTING_SESSION_ONLY:
+    case CONTENT_SETTING_BLOCK_THIRD_PARTY:
     case CONTENT_SETTING_DETECT_IMPORTANT_CONTENT:
     case CONTENT_SETTING_DEFAULT:
     case CONTENT_SETTING_NUM_SETTINGS:
